tests/test_span_session.rb

```diff
@@ -0,0 +1,99 @@
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/span_session'
+require_relative '../lib/cisco_node_utils/interface'
+require_relative '../lib/cisco_node_utils/vlan'
+require_relative '../lib/cisco_node_utils/cisco_cmn_utils'
+
+include Cisco
+
+# TestSpanSession - Minitest for SPAN session node utility
+class TestSpanSession < CiscoTestCase
+  @skip_unless_supported = 'session'
+  @@cleaned = false # rubocop:disable Style/ClassVars
+
+  def setup
+    super
+    cleanup unless @@cleaned
+    @@cleaned = true # rubocop:disable Style/ClassVars
+  end
+
+  def teardown
+    cleanup
+  end
+
+  def cleanup
+    SpanSession.sessions.each do |_session, obj|
+      obj.destroy
+    end
+  end
+
+  def test_create_new_session
+    span = SpanSession.new(1)
+
+    assert_equal(span.session_id, 1)
+    assert_equal(span.type, 'local') # default session type
+    assert(span.shutdown)
+    span.destroy
```

Comment:
You can actually remove the `span.destroy` lines that you have because the `teardown` method is automatically called after each test is run.